cli/internal/nodes/packagetask.go

```diff
@@ -21,10 +19,7 @@ type PackageTask struct {
 	Outputs         []string
 	ExcludedOutputs []string
 	LogFile         string
-	ExpandedInputs  map[turbopath.AnchoredUnixPath]string
 	Hash            string
-	HashedEnvVars   env.DetailedMap
-	Framework       string
```

Comment:
these pieces were originally added just to pass on to a task summary. we don't need them anymore